tests/regression/76-memleak/02-simple-mem-leak.c

```diff
@@ -0,0 +1,8 @@
+//PARAM: --set ana.malloc.unique_address_count 1 --set ana.activated[+] memLeak
+#include <stdlib.h>
+
+int main(int argc, char const *argv[]) {
+    int *p = malloc(sizeof(int));
+    // No free => memory is leaked
+    return 0; //TODO: `make test` detects OTHER and not WARN here
```

Comment:
Did you inspect what causes the test to report other? Maybe by tweaking the printing options a little bit, we can get it to produce the warning we want?